app/bundles/DynamicContentBundle/Resources/views/SubscribedEvents/Timeline/index.html.twig

```diff
@@ -1,7 +1,10 @@
 {% set data = event.extra.stat.sentDetails %}
+
 {% if data.failed is defined or data.timeline is not defined %}
```

Comment:
Shouldn't we change this line instead of creating an empty if part?
```suggestion
{% if data.failed is not defined and data.timeline is defined %}
```

Reply:
Sounds fair